crates/node_binding/src/lib.rs

```diff
@@ -112,7 +113,7 @@ unsafe impl<K, V> Send for SingleThreadedHashMap<K, V> {}
 unsafe impl<K, V> Sync for SingleThreadedHashMap<K, V> {}
 
 static COMPILERS: Lazy<
-  SingleThreadedHashMap<CompilerId, rspack_core::Compiler<AsyncNodeWritableFileSystem>>,
+  SingleThreadedHashMap<CompilerId, Pin<Box<rspack_core::Compiler<AsyncNodeWritableFileSystem>>>>,
```

Comment:
Could you add a comment to describe the SAFETY warning here?

Reply:
Done